pinot-common/src/main/java/org/apache/pinot/common/utils/DataSchema.java

```diff
@@ -215,7 +236,34 @@ public int hashCode() {
   }
 
   public enum ColumnDataType {
-    INT, LONG, FLOAT, DOUBLE, STRING, BYTES, OBJECT, INT_ARRAY, LONG_ARRAY, FLOAT_ARRAY, DOUBLE_ARRAY, STRING_ARRAY;
+    INT,
+    LONG,
+    FLOAT,
+    DOUBLE,
+    BOOLEAN /* Stored as INT */,
```

Comment:
Curious, do we plan other int types in the future? e.g. tinyint, short, int etc.

Reply:
Probably no. It doesn't make sense to add them as logical types and backed by int because that won't save any storage.